src/pages/login.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { navigate } from 'gatsby'
+import Layout from 'components/Layout'
+import SEO from 'components/Shared/Seo'
+import { connect } from 'react-redux'
+import { handleLogin } from 'store/actions/loginActions'
+import { isLoggedIn } from 'utils/auth'
+
+export const Login = ({ dispatch, loginReducer }) => {
+  const message = (isLoggedIn(loginReducer)) ? (<p>Hi {loginReducer.user.fullname}</p>) : ''
+
+  return (
+    <Layout>
+      <SEO title='Login' />
+      <h1>Login</h1>
```

Comment:
Let's get the title out of the main element. Change to:
```
<Layout
  title='Login'
  preMain={<SEO title='Login' />}
>
```

Reply:
done

Reply:
fix test